plugins/de/opendiabetes/vault/plugin/exporter/vaultodv/VaultODVExporter.java

```diff
@@ -32,20 +32,16 @@
 import java.security.DigestOutputStream;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.Date;
 import java.util.List;
 import java.util.Properties;
 import java.util.logging.Level;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipOutputStream;
 
-import static java.lang.Boolean.parseBoolean;
-
 /**
  * Not implemented yet.
  * Only a dummy class for the importer.
+ * TODO: might be superseded by the ODVExporter, will have to discuss with Jens about removal
```

Comment:
CheckStyle: Kommentar entspricht to-do-Format 'TODO:'.

Reply:
_will have to discuss with Jens about removal_
Absicht ;)

Reply:
Schon klar CheckStyle bemängelt nur dass das nicht das common TODO ist weil es ein ":" dahinter hat :D

Reply:
fixed